lib/Runtime/Library/JSONScanner.h

```diff
@@ -20,9 +20,16 @@ namespace JSON
             ::Js::ScriptContext* sc, const char16* current, ArenaAllocator* allocator);
 
         void Finalizer();
-        char16* GetCurrentString(){return currentString;}
-        uint GetCurrentStringLen(){return currentIndex;}
+        char16* GetCurrentString() { return currentString; } 
+        uint GetCurrentStringLen() { return currentIndex; }
+        uint GetScanPos() { return uint(currentChar - inputText); }
 
+        void ThrowSyntaxError(int wErr)
```

Comment:
the function takes a int32 hcode

err_method(ScriptContext* scriptContext, int32 hCode, PCWSTR varName = nullptr)